vite.content.config.ts

```diff
@@ -0,0 +1,33 @@
+import { resolve } from 'path'
+import { defineConfig } from 'vite'
+import getPagesViteConfig from './vite.config'
+
+const isDev = process.env.NODE_ENV !== 'production'
+const r = (...args: string[]) => resolve(__dirname, ...args)
+
+export default defineConfig(({ command }) => {
+  // @ts-expect-error
+  return Object.assign(getPagesViteConfig(command), {
```

Comment:
Use `mergeConfig` from `vite` instead

Reply:
I try mergeConfig , but have problems with build.rollupOptions.input . 

so i extract sharedConfig in vite.config.ts